### PR TITLE
Feat presentify

### PR DESCRIPTION
**Pull request by franz-monaco**

### CODE REVIEW

File: app/mdx-page/page.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
   return (
-    <div>
-      <div>
-        <Welcome />
-      </div>
+    <div className="flex flex-col">
+      {pages.map((page) => {
+        console.dir(page);
+        return (
+          <div className="w-full h-screen">
```
Comment (franzus85) [inline app/mdx-page/page.js:14]: key missing

File: app/mdx-page/page.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
 "use client";
 import Welcome from "@/markdown/welcome.mdx";
+import Welcome2 from "@/markdown/welcome2.mdx";
+import style from "./page.module.css";
+import { useMDXComponents } from "@mdx-js/react";
 
 export default function Page() {
+  const pages = [Welcome, Welcome2];
```
Comment (franzus85) [inline app/mdx-page/page.js:12]: read pages maybe via fs?

File: app/mdx-page/page.js (hunk 1 above)
Comment (franzus85) [inline app/mdx-page/page.js:14]:
```suggestion
       delete <div className="w-full h-screen">
```

File: app/mdx-page/page.module.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.content {
+  position: relative;
+  left: 50%;
+  top: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
```
Comment (franzus85) [inline app/mdx-page/page.module.css:6]:
```suggestion
  @apply mx-auto min-h-screen w-[1024px]
```

File: app/mdx-page/page.js (hunk 1 above)
Comment (franzus85) [inline app/mdx-page/page.js]:
```suggestion
```

Comment (franzus85) [reply app/mdx-page/page.js]: adapt page.module.css

File: app/mdx-page/page.js [hunk 4]
```diff
@@ -1,12 +1,23 @@
...
-    <div>
-      <div>
-        <Welcome />
-      </div>
+    <div className="flex flex-col">
+      {pages.map((page) => {
+        console.dir(page);
+        return (
+          <div className="w-full h-screen">
+            <div className={`${style.content} w-[1024px]`}>
```
Comment (franzus85) [inline app/mdx-page/page.js:19]: add key

File: app/mdx-page/layout.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import style from "./mdxlayout.module.css";
+
+export default function MdxLayout({ children }) {
+  // Create any shared layout or styles here
+
+  return (
+    <div style={{}} className="h-screen text-red-50" id={style.markdownLayout}>
```
Comment (franzus85) [inline app/mdx-page/layout.js:7]: style={}? and id=style?

**Review by franzus85 — CHANGES_REQUESTED** [review]
- css not quite understandable
- bold not working anymore

File: app/mdx-page/layout.js (hunk 5 above)
Comment (franz-monaco) [reply app/mdx-page/layout.js:7]: done

File: app/mdx-page/page.js (hunk 4 above)
Comment (franz-monaco) [reply app/mdx-page/page.js:19]: done

File: app/mdx-page/page.js (hunk 1 above)
Comment (franz-monaco) [reply app/mdx-page/page.js]: Why would it be better to put it in page.module.css rather then putting it as class?

File: app/mdx-page/page.module.css (hunk 3 above)
Comment (franz-monaco) [reply app/mdx-page/page.module.css:6]: - I get a different result, if i use your suggestion. I startet with that one too, but it does not center the slide properly.
- Bold was added again.

File: app/mdx-page/page.js (hunk 2 above)
Comment (franz-monaco) [reply app/mdx-page/page.js:12]: The effort is far too high. I would start with a static approach first.

**Comment by franz-monaco** [comment]
Code was moved to separate repository

**Pull request closed without merging**